.github/workflows/dbt-checks.yaml

```
@@ -0,0 +1,53 @@
+      - name: Setup DBT profile
+        working-directory: queries
+        run: sh setup_dbt_profiles.sh
+
+      - name: Run DBT compile
+        working-directory: queries
+        run: |-
+          dbt compile --profiles-dir profiles
+
+      - name: Run sqlfmt
+        working-directory: queries
+        run: sqlfmt --diff .
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)